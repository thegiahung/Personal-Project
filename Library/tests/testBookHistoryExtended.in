book history 111115
book history 111115 long
add book books.csv 111111
add member Java
add member Python
rent 100000 111111
relinquish 100000 111111
book history 111111
add collection books.csv
book history 111112
rent 100000 111112
relinquish 100000 111112
rent 100001 111112
relinquish 100001 111112
book history 111112
exit
