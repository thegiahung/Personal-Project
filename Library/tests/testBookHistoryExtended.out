user: No such book in system.

user: No such book in system.

user: Successfully added: To Kill a Mockingbird (Harper Lee).

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000

user: 16 books successfully added.

user: No rental history.

user: Success.

user: Success.

user: Success.

user: Success.

user: 100000
100001

user: Ending Library process.
